lib/widgets/breez_avatar.dart

```diff
@@ -91,46 +98,32 @@ class _GeneratedAvatar extends StatelessWidget {
 class _FileImageAvatar extends StatelessWidget {
   final double radius;
   final String filePath;
+  final bool isPreview;
 
-  const _FileImageAvatar(this.radius, this.filePath);
+  const _FileImageAvatar(this.radius, this.filePath, {this.isPreview = false});
 
   @override
   Widget build(BuildContext context) {
     return CircleAvatar(
       radius: radius,
-      backgroundColor: Colors.yellow,
       child: ClipRRect(
         borderRadius: BorderRadius.circular(radius),
-        child: FutureBuilder(
-          future: _getAvatarImageFile(),
-          builder: (BuildContext context, AsyncSnapshot<File> snapshot) {
-            if (snapshot.hasData) {
-              return Image(
-                image: FileImage(snapshot.data!),
-              );
-            }
-            return const SizedBox.shrink();
-          },
-        ),
+        child: isPreview
+            ? Image.file(
+                io.File(filePath),
+              )
+            : FutureBuilder<File?>(
+                future: DefaultCacheManager().getSingleFile(profileImageCacheKey),
```

Comment:
We need to check the performance of this when scrolling. Is the rendering smooth?

Reply:
I didn't experience any visible performance issues on debug mode with a payment list of ~50 items.